packages/sdk/wallets/wallet-walletconnect/typedoc.json

```diff
@@ -0,0 +1,13 @@
+{
+    "mode": "modules",
+    "exclude": ["**/*+(index|.test).ts"],
+    "excludeNotExported": true,
+    "excludePrivate": true,
+    "excludeProtected": true,
+    "includeDeclarations": false,
+    "ignoreCompilerErrors": true,
+    "hideGenerator": "true",
+    "theme": "gitbook",
+    "out": "../../../docs/developer-resources/wallet-rpc/reference",
```

Comment:
I think this would be its own directory following the [pattern of the other wallets](https://github.com/celo-org/celo-monorepo/tree/master/packages/docs/developer-resources):
`../../../docs/developer-resources/wallet-walletconnect/reference`